### PR TITLE
ISSUE_TEMPLATE, PULL_REQUEST_TEMPLATE guidelines added

### PR DESCRIPTION
**Pull request by optimistanoop**
Code of conduct , issue template, contributing and pull request template guidelines are among 6 key features for community profile for our project.

Ref: https://github.com/UdacityMobileWebScholarship/udabot/community

This PR will help our project stand out in terms of recommended community standards.
Templates provided in the PR will also help our contributors while creating PR and issues with standard guidelines by showing them the template as the last time checklist template in the description text area provided.

### CODE REVIEW

**Comment by optimistanoop** [comment]
@anshumanv I will do it now.

**Comment by optimistanoop** [comment]
@anshumanv It's done now.

File: .github/ISSUE_TEMPLATE.MD [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+#### How can the reviewer team reproduce the problem?
+
+<!-- Give a complete description of how to reproduce the problem. -->
+
+### supporting information:
+
+<!-- The following information MUST be included. -->
+
+ - [ ] `node -v` prints:
```
[the remark this answers is not in the document]
Comment (optimistanoop) [reply .github/ISSUE_TEMPLATE.MD]: @Biboswan Thanks for review, Please suggest - what / How should I change? What should be 
the required text ? Should it be like "please include following if it is front end issue"?

[the remark this answers is not in the document]
Comment (Biboswan) [reply .github/ISSUE_TEMPLATE.MD]: - [ ]  value of `node -v` (if it's not a UI related issue) :

[the remark this answers is not in the document]
Comment (optimistanoop) [reply .github/ISSUE_TEMPLATE.MD]: @Biboswan done, thanks.

**Pull request merged**